test/com/esotericsoftware/yamlbeans/YamlReaderTest.java

```diff
@@ -159,7 +159,8 @@ private Test read (String yaml) throws Exception {
 		if (true) {
 			System.out.println(yaml);
 			System.out.println("===");
-			System.out.println(new YamlReader(yaml).read(null));

```

Comment:
This line was not compiling on Eclipse Mars.2, saying that it was ambiguous. I updated it to this, which should have an equivalent behavior.